roofit/roofitcore/src/RooAbsData.cxx

```diff
@@ -346,15 +346,16 @@ void RooAbsData::setDirtyProp(Bool_t flag)
 /// Create a reduced copy of this dataset. The caller takes ownership of the returned dataset
 ///
 /// The following optional named arguments are accepted
-///
-///  - `SelectVars(const RooArgSet& vars)` Only retain the listed observables in the output dataset
-///  - `Cut(const char* expression)` Only retain event surviving the given cut expression
-///  - `Cut(const RooFormulaVar& expr)` Only retain event surviving the given cut formula
-///  - `CutRange(const char* name)` Only retain events inside range with given name. Multiple CutRange
+/// <table>
+/// <tr><td> SelectVars(const RooArgSet& vars)  <td> Only retain the listed observables in the output dataset
```

Comment:
maybe this first column can be enclosed between reverse quotes?

Reply:
Let's see if my skills with sed can save me tons of work ...

Reply:
Ok, call me regex master!